app/services/legacy/organisation_import_service.rb

```diff
@@ -6,11 +6,7 @@ def initialize(legacy_organisation)
   end
 
   def run
-    if organisation.persisted?
-      organisation.update_columns(organisation_attributes)
-    else
-      organisation.update_attributes(organisation_attributes)
-    end
+    organisation.update_attributes(organisation_attributes)
```

Comment:
@laurentqro - should we be using ! here and logging the exception?

Reply:
could do, yeah